go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/coreos/go-systemd
```

Comment:
I think we need to introduce the `/v21` here, no?

Reply:
Ah yes, I'll change that and adapt the imports.